Rakefile

```diff
@@ -7,14 +7,24 @@ namespace :collector do
     EventCollector.new.run
   end
 
-  desc 'run as daemon, collecting Zendesk, PagerDuty data periodically and updating REDIS db'
-  task :daemon do
-    require_relative 'services/event_collector.rb'
-    EventCollector.new.run_as_daemon
+  namespace :daemon do
+    desc 'run as daemon, collecting Zendesk, PagerDuty data periodically and updating REDIS db'
+    task :start do
+      system('ruby event_collector_control.rb start')
+    end
+
+    desc 'stop daemon'
+    task :stop do
+      system('ruby event_collector_control.rb stop')
+    end
+
+    desc 'daemon status'
+    task :status do
+      system('ruby event_collector_control.rb status')
```

Comment:
Switch these to exec so we don't end up with the rake process sat around.

And please document the reason why we can't call the ruby code directly.